[2022-09-05 03:07:29,366] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2022-08-04T00:00:00+00:00 [queued]>
[2022-09-05 03:07:29,481] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2022-08-04T00:00:00+00:00 [queued]>
[2022-09-05 03:07:29,481] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 03:07:29,482] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2022-09-05 03:07:29,485] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 03:07:29,608] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2022-08-04T00:00:00+00:00
[2022-09-05 03:07:29,615] {standard_task_runner.py:52} INFO - Started process 237 to run task
[2022-09-05 03:07:29,648] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v2', 'second_task', '2022-08-04T00:00:00+00:00', '--job-id', '22', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpd7d_jugc', '--error-file', '/tmp/tmp3kz4727m']
[2022-09-05 03:07:29,675] {standard_task_runner.py:77} INFO - Job 22: Subtask second_task
[2022-09-05 03:07:30,025] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v2.second_task 2022-08-04T00:00:00+00:00 [running]> on host 2b8e628fc16c
[2022-09-05 03:07:30,399] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-08-04T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-04T00:00:00+00:00
[2022-09-05 03:07:30,409] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-09-05 03:07:30,411] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2022-09-05 03:07:30,443] {bash.py:169} INFO - Output:
[2022-09-05 03:07:30,451] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2022-09-05 03:07:30,459] {bash.py:177} INFO - Command exited with return code 0
[2022-09-05 03:07:30,668] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20220804T000000, start_date=20220905T030729, end_date=20220905T030730
[2022-09-05 03:07:31,011] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-09-05 03:07:31,074] {local_task_job.py:146} INFO - Task exited with return code 0
